libs/platform/src/lib/components/form/multi-combobox/multi-combobox/multi-combobox.component.scss

```diff
@@ -0,0 +1,32 @@
+@import '~@angular/cdk/overlay-prebuilt';
+@import '~fundamental-styles/dist/margins';
+
+.fdp-multi-combobox-tokenizer-custom {
+    width: calc(100% - 2.25rem);
+    &--compact {
+        width: calc(100% - 2rem);
+    }
+}
+
+.fdp-multi-combobox__list-container {
+    .fd-list {
+        .fd-list__item {
+            &.fd-list__group-header {
+                padding: 0 1rem;
+            }
+
+            .fd-list__title {
+                // TODO: to investigate
```

Comment:
Could you please provide more info in comment (in code, not here) about that TODO item? Thanks